docs/ansible/ansible.md

```diff
@@ -7,6 +7,8 @@ Depending on your available python version you may be limited in choosing which
 
 It is recommended to deploy the ansible version used by kubespray into a python virtual environment.
 
+Run below commands under kubespray git directory:
+
```

Comment:
I think this sentence is a bit redundant and misrepresented because these commands are not run in the Kubespray git directory.